outlook/mail/tool.gpt

```diff
@@ -103,26 +103,18 @@ Param: destination_folder_id: The ID of the folder to move the message into.
 
 #!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool moveMessage
 
----
-Name: Get Default Timezone
-Description: Get the default timezone for the user.
-Credential: ./credential
-
-#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool getDefaultTimezone
-
 ---
 Name: Outlook Mail Context
 Type: context
-Share Context: Current Date and Time from ../../time
-Share Tools: Get Default Timezone
+Share Context: ../../time
 
 #!sys.echo
 
-## Instructions for using the Microsoft Outlook Mail tools
+# START INSTRUCTIONS: Microsoft Outlook Mail tools
```

Comment:
nit: i think this change is fighting against the established norm.

Reply:
There's precedent in several other tools AFACT https://github.com/search?q=repo%3Aobot-platform%2Ftools+%22START+INSTRUCTIONS%22&type=code

Reply:
I dropped this change